Coding/fw/lib/Settings/WLAN.cpp

```diff
@@ -49,7 +49,7 @@ static bool connectWiFi();
 
 /* VARIABLES **************************************************************************************/
 IPAddress m_serverIP; /**< Stores the IP Address of the ESP32 */
-bool m_APMode = true;
+static bool m_APMode = true;
```

Comment:
add comment so APMode variable, 
Why do these variables have an m_ prefix ? 
Why is serverIP global not static  (see const IPAddress &wlan::getIPAddress() for a getter ) ?